Create ui-c8y-1022-15-0-view-the-details-of-a-device-parameter-graft-release-cd.md (#3626)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-15-0-view-the-details-of-a-device-parameter-graft-release-cd.md

---
date: ""

> title: Details pane added to the device parameters tab

Reviewer: ```suggestion
title: Details on device parameters available on the Parameters tab
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4667
version: 1022.15.0
---

> In **Device management**, users can now view the details of a device parameter in the **Parameters** tab. Next to the parameter description and current value users can view the history of value change operations, as well as the history of value change events.

Reviewer: ```suggestion
In the **Parameters** tab in the device details, details on device parameters are now provided. Next to the parameter description and current value, users can view the history of value change operations, as well as the history of value change events.
```

> In the **Parameters** tab in the device details, details on device parameters are now provided. Next to the parameter description and current value, users can view the history of value change operations, as well as the history of value change events.

Reviewer: ```suggestion
On the **Parameters** tab in the device details, details on device parameters are now provided. Next to the parameter description and current value, users can view the history of value change operations, as well as the history of value change events.
```

---
date: ""

> title: View the details of a device parameter [GRAFT][release/cd]

Reviewer 2: ```suggestion
title: Details pane added to the device parameters tab
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4667
version: 1022.15.0
---
Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>
Co-authored-by: Dafina Karamanoleva <dafina.karamanoleva@cumulocity.com>

> Co-authored-by: Dominik Schmitz <dominik.schmitz@cumulocity.com>

Reviewer 2: ```suggestion
In **Device management**, users can now view the details of a device parameter in the **Parameters** tab. Next to the parameter description and current value users can view the history of value change operations, as well as the history of value change events.
```
